<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Picframe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>102</x>
      <y>83</y>
      <width>531</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string>导入目录：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Input_path">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string>暂未选择导入目录</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Input_Button">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string>导出目录：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="Output_path">
         <property name="text">
          <string>暂未选择导出目录</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Output_Button">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string>选择</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>字体设置：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFontComboBox" name="fontComboBox"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>选择字体</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>字体缩放：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
       </item>
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>倍</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_13">
         <property name="text">
          <string>字体颜色：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox">
         <item>
          <property name="text">
           <string>黑色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>红色</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>暗夜紫(92,34,35)</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string>输出质量：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="quality">
         <property name="minimum">
          <number>40</number>
         </property>
         <property name="maximum">
          <number>100</number>
         </property>
         <property name="singleStep">
          <number>10</number>
         </property>
         <property name="value">
          <number>80</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_11">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string>命名方式：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="renameBox">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <item>
          <property name="text">
           <string>Blank+原文件名</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>序列号</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="font">
          <font>
           <family>MiSans Demibold</family>
           <pointsize>12</pointsize>
           <bold>true</bold>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>开始</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="text">
          <string>正在处理：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="filename">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string> </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_12">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string>剩余数目：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="remain_num">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string> </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string>制造商：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="make">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string> </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_10">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string>相机型号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="model">
         <property name="font">
          <font>
           <family>MiSans</family>
          </font>
         </property>
         <property name="text">
          <string> </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="detail_label">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>处理详情</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>进度：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
